docusaurus/docs/guides/securing-apis/aks-api-with-nginx-ziti-module.md

```diff
@@ -43,7 +43,7 @@ client name = `client-nginx` with `Attribute`: `#clients`,  server module name =
 
 Download jwt files and enroll identities. 
 
-- Windows ZDE Identity can be enrolled by following this [enrolling process](https://docs.openziti.io/docs/reference/tunnelers/windows#enrolling)
+- Windows ZDE Identity can be enrolled by following this [enrolling process](../../../docs/reference/tunnelers/02-windows.md#enrolling-)

```

Comment:
```suggestion
- Windows ZDE Identity can be enrolled by following this [enrolling process](../../../docs/reference/tunnelers/02-windows.md#enrolling)
```

Reply:
fixes a broken fragment link

Reply:
golang put that there and i didn't bother to verify it. I tried it out and I see that it works fine but the `-` seems to 'hide the title': https://ziti-doc-git-remove-docs-refs-openziti.vercel.app/docs/reference/tunnelers/windows#enrolling-

so i prefer without the `-`